release/macos/release.sh

```diff
@@ -157,3 +157,12 @@ if [ -r $NOT_KEY_ID_F ]; then
         $INSTALLER
     rm $NOT_JSON
 fi
+
+GPG_PRIVKEY=/release/gpg-privkey.asc
+if [ -r $GPG_PRIVKEY ]; then
+    export GPG_TTY=$(tty)
+    export LC_ALL=C.UTF-8
+    gpg --import /release/privkey.asc
+    GPGSIGNKEY=$(gpg --list-keys --with-colons | grep '^pub' | cut -d ":" -f 5)
+    gpg -u $GPGSIGNKEY --armor --detach-sig /release/build/*.pkg
```

Comment:
btw I am not exactly sure if we really need to GPG sign the macOS and windows releases when they are already signed and macOS even notarized, but well, we did it already so I just moved it here

Reply:
Yeah I agree, but we do that for Suite as well, so let's keep it for now.